### ISSUE TITLE
Standard Notes totally black windows on Linux

### ISSUE THREAD

**Issue by Manawyddan** [standardnotes/forum#3381]
**Describe the bug**
Whenever i start Standard Notes via cli or just click to open de app, it opens but the window is just totally black, if i move the mouse cursor around it changes according to the options available on the several areas of the apps window, but it's always completely black, can't see what i'm doing.

Sending images attached.

![imagem](https://github.com/standardnotes/forum/assets/33188344/c2de3ddd-5f3f-4246-b0bc-db36923aa979)
![Screenshot_2023-11-27_23_16_47](https://github.com/standardnotes/forum/assets/33188344/7654f44d-17f1-44bb-8259-5667e9a89b3b)
![Screenshot_2023-11-27_23_16_53](https://github.com/standardnotes/forum/assets/33188344/1f49eaea-e1c9-4a5d-9a11-3f3aba4857cc)
![Screenshot_2023-11-27_23_16_58](https://github.com/standardnotes/forum/assets/33188344/65b175da-6ea7-44a2-bec1-bc500564ac7e)

**Desktop (please complete the following information):**
 - Kali Linux(6.5.0-kali3-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.5.6-1kali1 (2023-10-09) x86_64 GNU/Linux)

**Additional context**

Any idea what might be causing this issue?

Thank you very much,

Kind regards,

**Comment by effieeee** [comment]
Hi @Manawyddan, may I know whether you're using the **AppImage**, **Snap**, or **deb** package? Would you also happen to recall what version of the app is running on your device at the moment?

**Comment by Manawyddan** [comment]
Hi @effieeee,

My apologies, totally forgot to provide that information.
I'm using the AppImage version of the Standard Notes App, using version standard-notes-3.181.23-linux-x86_64.AppImage at the moment.
I also wanted to inform that it is now working properly again.
What I did was I closed the app, then removed all the config files and folders relevant to the Standard Notes app on my /home/user folder, and then I started the app again, config folders and files created anew on my /home/user directory and it's now working again.
Sorry for the trouble, it was an easy fix after all.

Thank you very much.

Kind regards,
